main.go

```diff
@@ -68,26 +73,50 @@ func hello(c echo.Context) error {
 	return c.String(http.StatusOK, "Hello, World!")
 }
 
-// user
-// @Summary Add a new User
-// @Description Adds a new user to the list of Users
-// @ID add-user
+// OF Trace
+// @Summary Send trace request to OpenFisca
+// @ID of-trace
 // @Accept  json
 // @Produce  json
-// @Success 200 {string} string	"Returns new user"
-// @Router /user [post]
-func user(c echo.Context) error {
-	// Create a new user
-	u := new(User)
-	// Bind the request body to the new user
-	if err := c.Bind(u); err != nil {
-		return err
+// @Success 200 {string} string	"Returns OpenFisca trace response"
+// @Router /trace [post]
+func trace(c echo.Context) (err error) {
+
+	data := c.Request().Body
+
+	body, err := ioutil.ReadAll(data)
+	if err != nil {
+		log.Printf("Error reading body: %v", err)
+		// http.Error("can't read body", http.StatusBadRequest)
+		return
+	}
+
+	req, err := http.NewRequest("POST", "http://localhost:5000/trace", bytes.NewBuffer(body))
+	if err != nil {
+		fmt.Println("Error in req: ", err)
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	// Create a Client
+	client := &http.Client{}
```

Comment:
For future enhancement, creating a new client to call the same endpoint should be avoided if possible. It will be fine for now but we should look at creating a client pool and reusing connections to be more efficient and still be able to handle large amounts of simultaneous requests